gui/src/test/java/org/verapdf/cli/VeraPdfCliProcessorTest.java

```diff
@@ -95,34 +95,6 @@ public final void testCreateProcessorFromArgsLogPassed()
      * @throws ProfileException
      */
     @Test
-    public final void testCreateProcessorFromArgsExtract()
```

Comment:
you should not remove this test. Tests can be removed only if some logic has been removed. in this place you just moved arg -x to subset of another art (-t validatingAndFeatures).